core/account-id/README.md

```diff
@@ -0,0 +1,69 @@
+# near-account-id
+
+This crate provides a type for representing a valid, unique account identifier on the [NEAR](https://near.org) network.
+
+[![crates.io](https://img.shields.io/crates/v/near-account-id?label=latest)](https://crates.io/crates/near-account-id)
+[![Documentation](https://docs.rs/near-account-id/badge.svg)](https://docs.rs/near-account-id)
+[![Version](https://img.shields.io/badge/rustc-1.56+-ab6000.svg)](https://blog.rust-lang.org/2021/10/21/Rust-1.56.0.html)
```

Comment:
updating this feels like a chore. I sugest *not* having this and instead rely on `rust-version` in Cargo.toml

Reply:
My thinking was this wasn't going to change very often, since it's unlikely this crate changes a lot once stabilized. I'll drop it nonetheless.